src/routes/pages/Bookmark.tsx

```diff
@@ -0,0 +1,147 @@
+import { useEffect, useRef, useState } from 'react'
+import Category from '@/components/common/Category'
+import EmptyInfo from '@/components/emptyInfo/EmptyInfo'
+import BookmarkItem from '@/components/bookmarks/BookmarkItem'
+import { useFetchPlaylists } from '@/hooks/useFetchPlaylists'
+import { useFetchUserBookmark } from '@/hooks/useFetchUserBookmark'
+import { useHeaderStore } from '@/stores/header'
+import { css } from '@emotion/react'
+import Sortable from 'sortablejs'
+import Toast from '@/components/common/Toast'
+
+const Bookmark = () => {
+  const setTitle = useHeaderStore(state => state.setTitle)
+  const { data: Playlists } = useFetchPlaylists()
+  const { data: bookmarkMap } = useFetchUserBookmark()
+  const ItemRef = useRef<HTMLDivElement | null>(null)
+
+  const [selectedCategories, setSelectedCategories] = useState<string[]>([
+    '전체'
+  ])
+  const [isToastVisible, setIsToastVisible] = useState(false)
+
+  useEffect(() => {
+    setTitle('북마크')
+  }, [setTitle])
+
+  useEffect(() => {
+    if (selectedCategories.length === 0) {
+      setSelectedCategories(['전체'])
+    }
+  }, [selectedCategories])
+
+  const filteredData =
+    selectedCategories.length > 0
+      ? Playlists?.filter(
+          pl =>
+            Array.isArray(pl.categories) &&
+            selectedCategories.some(cat => pl.categories.includes(cat))
+        )
+      : Playlists
+
+  // playlistId와 bookmarkTimestamp를 이용하여 북마크 시점으로 최신 정렬
+  const filteredPlaylists = filteredData
+    ?.filter(pl => bookmarkMap?.has(pl.id))
+    ?.sort((a, b) => {
```

Comment:
P3;

요건 ?를 빼도 되지 않을까요?
```suggestion
    .sort((a, b) => {
```

Reply:
넵 뺐습니다 감사합니다!